### ISSUE TITLE
The color of icon blends with the Dock when the dark theme is on.

### ISSUE THREAD

**Issue by lk-geimfari** [mullvad/mullvadvpn-app#3682]
## Issue report

Operating system: macOS Big Sur

App version: 2022.2

### Issue description

The color of the icon of Mullvad blends with the Dock when the dark theme turns on.

This is how it looks almost on all default dark themes:

<img width="447" alt="Screenshot 2022-06-22 at 02 45 40" src="https://user-images.githubusercontent.com/15812620/174914465-7f022c27-92f6-4ef2-a5b3-84a0067fccd0.png">
<img width="402" alt="Screenshot 2022-06-22 at 02 51 43" src="https://user-images.githubusercontent.com/15812620/174915042-cbf21598-0202-4e2b-9b2c-d85e3baf94f4.png">
<img width="413" alt="Screenshot 2022-06-22 at 02 52 09" src="https://user-images.githubusercontent.com/15812620/174915056-70a2d9dc-ccf5-4cfb-811e-2f4a1be237b6.png">

I know that this is probably not super priority issue for you, but Mullvad is absolutely outstanding service, so I beilive this tine UI fix make user expirence better.

**tl;dr** having icon for dark mode would be a great addition for Mullvad.

**Comment by faern** [comment]
Thanks for reporting! Indeed fairly minor, the mole itself is clearly visible. We'll keep the issue open and may look at it eventually.
